substrate/runtime/src/lib.rs

```diff
@@ -290,7 +295,7 @@ pub type ReportLongevity = <Runtime as pallet_babe::Config>::EpochDuration;
 
 impl babe::Config for Runtime {
   #[cfg(feature = "fast-epoch")]
-  type EpochDuration = ConstU64<{ MINUTES / 2 }>; // 30 seconds
+  type EpochDuration = ConstU64<{ HOURS / 2 }>; // 30 minutes
```

Comment:
lol. It was set to be 30 min because I knew it was like this in the previous pr. So I wanted to make sure the test passes with the same epoch time. But apparently it works just as good with 30 sec so we don't need this change 👍